webpack.common.config.js

```diff
@@ -153,7 +153,8 @@ module.exports = {
         },
         modules: [
             'node_modules',
-            'common/static/js/vendor/'
+            'common/static/js/vendor/',
+            'lms/static'
```

Comment:
This change I want extra scrutiny on -- I did this to help eslint find import lines.  But as noted elsewhere, I changed course_search's eslint config to look at the webpack config for how the code is layed out.

I think this change should be fine for the real webpack config too?  But I'm not sure, since I'm not familiar with webpack.